Cargo.toml

```diff
@@ -60,27 +60,21 @@ rev = "70b0eb5"
 # * Utilize `master` branch of smoltcp
 # * Utilize `master` branch of HAL
 [dependencies.stm32h7xx-hal]
-features = ["stm32h743v", "rt", "unproven", "ethernet", "quadspi"]
-# version = "0.9.0"
-git = "https://github.com/quartiq/stm32h7xx-hal.git"
-rev = "33aa67d"
+features = ["stm32h743v", "rt", "unproven", "ethernet", "xspi"]
+git = "https://github.com/stm32-rs/stm32h7xx-hal"
+branch = "v0.11.0"
```

Comment:
Let's wait till this gets released, should be pretty soon.

Reply:
Could we land it now anyways so we can collect some running time with the new code?

Reply:
Yeah, would be happy to land it. I'll update this to point to a rev.